### DISCUSSION TITLE
Caching JSON Data

### DISCUSSION THREAD

**Discussion by zebede22** [dragonflydb/dragonfly#2568] [General]
I already use dragonfly for pubsub, now I want to cache json data with it.

My servers generate json files and store them to Wasabi S3 servers, I plan to store them into dragonfly as well and use as a cache, but with some limitations, for example to store only 5GB and rotate data.

Is that possible?
Any guides?

**Comment by romange** [comment]
you can use `--cache_mode` flag together with `--maxmemory=5GB`. Having said that, 5GB can be too low for a limit.

**Reply by zebede22** [reply]
That just for an example, thanks!
